core/src/lib.rs

```diff
@@ -1,15 +1,22 @@
+#![allow(unused_variables)]
+#![allow(dead_code)]
 mod credential;
-
 use credential::*;
 use serde_json::{self, Value};
+use signature::keypair::Ed25519SSIKeyPair;
```

Comment:
We should generally refrain from `use` and instead utilize things through the appropriate namespace i.e `signature::keypair::Ed25519....`